src/services/uniswapV3/UniswapV3LiquidityResolver.ts

```diff
@@ -0,0 +1,110 @@
+import {inject, injectable} from 'inversify';
+import {Logger} from 'winston';
+import {Token, ChainId} from '@uniswap/sdk-core';
+
+import TimeService from '../TimeService.js';
+import {FeedsType} from '../../types/Feed.js';
+import {FeedDataService} from '../FeedDataService.js';
+import LiquidityService from './LiquidityService.js';
+import {LeavesAndFeeds} from '../../types/Consensus.js';
+import {BlockchainProviderRepository} from '../../repositories/BlockchainProviderRepository.js';
+import {PoolRepository} from '../../repositories/PoolRepository.js';
+import {ChainsIds} from '../../types/ChainsIds.js';
+import {TokenRepository} from '../../repositories/TokenRepository.js';
+import {UniswapV3Param} from './interfaces.js';
+
+@injectable()
+export class UniswapV3LiquidityResolver {
+  @inject(FeedDataService) feedDataService!: FeedDataService;
+  @inject(TimeService) timeService!: TimeService;
+  @inject(TokenRepository) tokenRepository!: TokenRepository;
+  @inject(LiquidityService) liquidityService!: LiquidityService;
+  @inject(BlockchainProviderRepository) blockchainProviderRepository!: BlockchainProviderRepository;
+  @inject(PoolRepository) poolRepository!: PoolRepository;
+  @inject('Logger') logger!: Logger;
+
+  constructor() {}
+
+  async apply() {
+    const dataTimestamp = this.timeService.apply();
+    const data = await this.feedDataService.apply(dataTimestamp, FeedsType.CONSENSUS);
+
+    if (data.rejected) {
+      this.logger.info(`[UniswapV3LiquidityResolver] rejected: ${data.rejected}`);
+    }
+
+    const uniswapV3Params = await this.feedDataService.getParamsByFetcherName<UniswapV3Param>(
+      data.feeds as LeavesAndFeeds,
+      'UniswapV3',
+    );
+
+    if (uniswapV3Params.length === 0) {
+      throw new Error('[UniswapV3LiquidityResolver] No params for fetcher');
+    }
+
+    const address = '0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8';
```

Comment:
Will this be hard coded? Maybe it could be set through the settings and be a class parameter?

Reply:
It will come from the Pool model, which I will change. Thanks